<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_forwarder</class>
 <widget class="QWidget" name="menu_forwarder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Manjari&quot;;
</string>
     </property>
     <property name="text">
      <string>FORWARDER MENU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>420</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>LOG OUT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>AVAILABLE CARGO ORDERS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>290</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>MY ORDERS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>PROGRESS CARGO</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Manjari&quot;;
</string>
     </property>
     <property name="text">
      <string>AVAILABLE CARGO'S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>GO BACK</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>571</width>
       <height>341</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>430</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>MAKE AN OFFER</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Manjari&quot;;
</string>
     </property>
     <property name="text">
      <string>PROGRESS CARGO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>GO BACK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>430</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>SEND OFFER</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>491</width>
       <height>251</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>370</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(85, 87, 83);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>350</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
</string>
     </property>
     <property name="text">
      <string>CHOOSE CARGO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
</string>
     </property>
     <property name="text">
      <string>CHOOSE COMPANY</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Manjari&quot;;
</string>
     </property>
     <property name="text">
      <string>ORDER HISTORY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Manjari&quot;;
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string>GO BACK</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>581</width>
       <height>421</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
